charts/pulsar/templates/broker/function-worker-configfile-configmap.yaml

```diff
@@ -106,6 +106,15 @@ data:
     brokerClientTrustCertsFilePath: "/pulsar/certs/ca/ca.crt"
     {{- end }}
 
+    {{- if .Values.functions.skipTls }}
```

Comment:
The name `skipTls` is a bit confusing, actually the traffic is still on TLS in this case, it just does not reject invalid certificates.

Reply:
updated to .Values.functions.tlsAllowInsecureConn